docs/sources/datasources/graphite.md

```diff
@@ -141,6 +141,27 @@ For more details, see the [Graphite docs on the autocomplete API for tags](http:
 The query you specify in the query field should be a metric find type of query. For example, a query like `prod.servers.*` will fill the
 variable with all possible values that exist in the wildcard position.
 
+The results will contain all possible values occurring only at the last level of the query. To get full metric names matching the query 
+use expand function (`expand(*.servers.*)`). 
+
+#### Comparison between expanded and non-expanded metric search results
+
+Example metrics:
+- `prod.servers.001.cpu`
+- `prod.servers.002.cpu`
+- `test.servers.001.cpu`
+
+| normal query | results | expanded query | expanded results |
+|--------------|---------|----------------|------------------|
+| `*` | prod, test | `expand(*)` | prod, test
+| `*.servers` | servers | `expand(*.servers)` | prod.servers, test.servers |
+| `test.servers` | servers | `expand(test.servers)` | test.servers |
+| `*.servers.*` | 001,002 | `expand(*.servers.*)` | prod.servers.001, prod.servers.002, test.servers.001 |
+| `test.servers.*` | 001 | `expand(test.servers.*)` | test.servers.001 |
+| `*.servers.*.cpu` | cpu | `expand(*.servers.*.cpu)` | prod.servers.001.cpu, prod.servers.002.cpu, test.servers.001.cpu |
+
+You can combine `expand` with a regex to match specific part of the metric name path.
```

Comment:
Remove, as we covered this in the intro section I have suggested?

Reply:
Makes sense. Removed.